### ISSUE TITLE
install_elk.sh is out of date

### ISSUE THREAD

**Issue by zw1068** [pwnlandia/mhn#744]
I run ' install_elk.sh '，when ‘apt-get install  -y python-software-properties
’，an error occurred：
```
+ mkdir -p /var/log/mhn
+ apt-get install -y supervisor
Reading package lists... Done
Building dependency tree       
Reading state information... Done
supervisor is already the newest version (3.3.1-1.1).
0 upgraded, 0 newly installed, 0 to remove and 8 not upgraded.
+ cat
+ supervisorctl update
+ apt-get install -y python-software-properties
Reading package lists... Done
Building dependency tree       
Reading state information... Done
Package python-software-properties is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source
However the following packages replace it:
  software-properties-common

E: Package 'python-software-properties' has no installation candidate
```

I run ‘ apt-get install software-properties-common’
```
root@ubuntu:/opt/mhn# apt-get install software-properties-common 
Reading package lists... Done
Building dependency tree       
Reading state information... Done
software-properties-common is already the newest version (0.96.24.32.12).
software-properties-common set to manually installed.
0 upgraded, 0 newly installed, 0 to remove and 8 not upgraded.

```
but,It dose not work.

**Comment by d1str0** [comment]
This is being addressed by PR #741 

Will test and hopefully merge soon.

**Comment by d1str0** [comment]
#741 has been merged
